extensions/dicom-video/src/getSopClassHandlerModule.js

```diff
@@ -72,20 +73,29 @@ const _getDisplaySetsFromSeries = (instances, servicesManager, extensionManager)
         SOPClassHandlerId,
         referencedImages: null,
         measurements: null,
+        viewportType: 'video',
+        // The videoUrl is deprecated, the preferred URL is renderedUrl
         videoUrl,
+        renderedUrl: videoUrl,
         instances: [instance],
         thumbnailSrc: dataSource.retrieve.directURL({
           instance,
           defaultPath: '/thumbnail',
           defaultType: 'image/jpeg',
           tag: 'Absent',
         }),
+        imageIds: [imageId],
         isDerivedDisplaySet: true,
         isLoaded: false,
+        isClip: true,
```

Comment:
The only other place we use `isClip` is in multiframe. I'm wondering if it's correct. Perhaps a comment in the code would clarify its purpose.

Reply:
Actually we don't reference it anywhere in the code, so just deleting it.